mapie/multi_label_classification.py

```diff
@@ -158,21 +170,21 @@ def _check_parameters(self) -> None:
         check_verbose(self.verbose)
         check_random_state(self.random_state)
 
-    def _check_method(self) -> None:
-        """
-        Check that the specified method is valid
-
-        Raises
-        ------
-        ValueError
-            Raise error if the name of the method is not
-            in self.valid_methods_
-        """
-        if self.method not in self.valid_methods_:
-            raise ValueError(
-                "Invalid method. "
-                "Allowed values are" + " or ".join(self.valid_methods_)
-            )
+    # def _check_method(self) -> None:
+    #     """
+    #     Check that the specified method is valid
+
+    #     Raises
+    #     ------
+    #     ValueError
+    #         Raise error if the name of the method is not
+    #         in self.valid_methods_
+    #     """
+    #     if self.method not in self.valid_methods_:
+    #         raise ValueError(
+    #             "Invalid method. "
+    #             "Allowed methods are " + " or ".join(self.valid_methods_)
+    #         )
```

Comment:
UPD: should be kept (refered to our previous conversation).

Reply:
Done